tools/x-repo-types/Makefile

```diff
@@ -0,0 +1,35 @@
+all: goal-v-sdk-state-delta-xrt goal-v-sdk-bookkeeping-genesis goal-v-sdk-stateproof goal-v-spv-stateproof
+
+template4xrt:
+	echo "// DO NOT MODIFY!!!! Template generated by 'make template4xrt'." > xrt_tmpl.go.tmpl
+	sed -e '/\/\* T/ s/^..../\t/' -e '/\/\* G/ s/^/\t\/\/ /' -e 's/Main()/main()/g' xrt_tmpl.go >> xrt_tmpl.go.tmpl
+
+build-xrt: template4xrt
+	go build -o xrt
+
+goal-v-sdk-state-delta-xrt:
```

Comment:
This is probably not the best place to keep an "official list" of comparison types, but maybe for now it's good 'nuf?

Reply:
`bookkeeping.Block` - If this doesn't, and it's the only failure, work let's discuss whether or not to address it.
`bookkeeping.BlockHeader` - One of the most common things we used to miss.

Reply:
This is now being run under `make -k all`